lib/Dialect/FIRRTL/Transforms/GrandCentralTaps.cpp

```diff
@@ -505,8 +519,7 @@ LogicalResult static applyNoBlackBoxStyleDataTaps(const AnnoPathValue &target,
 
     // Match previous behavior, don't use "good" names for dataflow wires.
     // (Store as a StringAttr so the wiringProblem StringRef is backed.)
-    auto nameHint =
-        StringAttr::get(context, Twine("_gen_") + tapName.getValue());
+    auto nameHint = StringAttr::get(context, tapName.getValue());
```

Comment:
The comment above this line is no longer accurate I don't think, probably no reason to make this a `StringAttr` (yet). Which means this can probably either just be replaced with `tapName` or `StringRef nameHint = ` or something.

Reply:
Yep, totally not necessary. `tapName` is _already_ a `StringAttr`.